vagrant/provisioning/vm_config.conf.yml

```diff
@@ -5,13 +5,13 @@ k8smaster:
   short_name: "k8smaster"
   host_name: "k8smaster.ovn"
   public-ip: "10.10.0.11"
-  memory: 2048
+  memory: 3192
   cpus: 2
 k8sminion1:
   short_name: "k8sminion1"
   host_name: "k8sminion1.ovn"
   public-ip: "10.10.0.12"
-  memory: 2048
```

Comment:
Unrelated change?

Reply:
Yeah, will revert it. BTW, Vagrant VM's 2048 memory isn't sufficient for daemonset mode. Some of the PODs don't start and remain in pending with 'insufficient memory' error.